### PR TITLE
:white_check_mark: reducing flakiness of tests by removing randomization

### PR DESCRIPTION
**Pull request by MatissJanis**
This is not a full fix for the flakiness. One of the test cases will still be flaky. But at least this fixes the other test cases thus improving stability.

### CODE REVIEW

File: packages/loot-core/src/mocks/index.js [hunk 1]
```diff
@@ -55,9 +55,8 @@ function _generateTransaction(data) {
   return {
     id: id,
     amount: data.amount || Math.floor(Math.random() * 10000 - 7000),
-    payee: data.payee || (Math.random() < 0.9 ? 'payed-to' : 'guy'),
-    notes:
-      Math.random() < 0.1 ? 'A really long note that should overflow' : 'Notes',
+    payee: data.payee || 'payed-to',
+    notes: 'Notes',
```
Comment (j-f1) [inline packages/loot-core/src/mocks/index.js:59]: Is there a way to test that these cases are still handled properly? (overflow)

**Review by j-f1 — APPROVED** [review]
LGTM, leaving one point that can be handled in the future / never

File: packages/loot-core/src/mocks/index.js (hunk 1 above)
Comment (MatissJanis) [reply packages/loot-core/src/mocks/index.js:59]: No. And we didn't have one before either.

Text overflow is a visual aspect of the product that should be tested with visual regression tests (screenshots). But we have none of those.

Comment (j-f1) [reply packages/loot-core/src/mocks/index.js:59]: Ok, sounds good!

**Pull request merged**